.github/workflows/test-coin-tester.yml

```diff
@@ -0,0 +1,128 @@
+name: "[Test] Coin Tester"
+
+on:
+  schedule:
+    - cron: "9 16 * * 1-5"
+
+  pull_request:
+    branches-ignore:
+      - "smartling-content-updated**"
+      - "smartling-translation-completed**"
+  workflow_dispatch:
+    inputs:
+      chain:
+        description: "Coin chain to test (coin name or leave blank for auto)"
```

Comment:
Maybe we can provide an example ? or list of options

If list is static we can also list options with a "choice" type

```
        type: choice
        description: packages to run
        required: false
        options: 
        - evm
        - polkadot
```

Reply:
change description to `Coin family to test, separated by commas (,)"` 
For now it's static but we will add solana BTC and others in the future